Project.toml

```diff
@@ -3,14 +3,14 @@ uuid = "123dc426-2d89-5057-bbad-38513e3affd8"
 version = "0.7.1"
 
 [deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
 Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
-Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+SymEngine_jll = "3428059b-622b-5399-b16f-d347a77089a4"
 
 [compat]
-BinaryProvider = "≥ 0.3.0"
 Compat = "≥ 0.63.0"
 RecipesBase = "≥ 0.0.6"
-julia = "≥ 1.0.0"
+julia = "≥ 1.3.0"
```

Comment:
Note that dependencies need to have upper bounds. This won't be accepted in the general registry

Reply:
Something like, `julia = "≥ 1.3.0,<2.0.0"?

Reply:
Simply
```
julia ="1.3"
```
also works